.gitignore

```diff
@@ -1,7 +1,3 @@
-/vendor
-
+/composer.lock
 /iso-codes
-
-/.vscode
```

Comment:
This is developer specific, are you going to do this for intelliJ, Zend Studio, Eclipse PDT etc? Being a developer's environment it's the developers responsibility to configure their system with `~/.gitgitnore` specific to their chosen tooling.

Reply:
As a project maintainer, you need to ensure no-one tries to commit any tooling specific files to your library